### PR TITLE
Add suite logger to capture each individual suite run

### PR DESCRIPTION
**Pull request by sersut**
/cc: @patrick-wright, @tyler-ball

### CODE REVIEW

File: lib/chef-acceptance/logger.rb [hunk 1]
```diff
@@ -2,36 +2,43 @@
...
+    attr_reader :file_logger
     attr_reader :stdout_logger
-    attr_reader :suite_loggers
 
-    def initialize(base_dir = Dir.pwd)
-      @log_directory = File.join(base_dir, ".acceptance_logs")
-      @suite_loggers = []
+    # Supported options:
+    # log_path: path
+    # log_header:
```
Comment (sersut) [inline lib/chef-acceptance/logger.rb]: We probably need a bit better documentation here.

**Comment by tyler-ball** [comment]
:+1:

**Comment by patrick-wright** [comment]
I think it would be useful to make an entry in the acceptance.log when suites/commands start.

The logs look really great.  Some newline things from kitchen make it look weird in places, but no big deal.

I would like to have another card for stdout changes for when things are logged.  I'm good with having  acceptance.log and stdout match for this PR, but it is a bit much.  Too much noise for stdout.

**Comment by tyler-ball** [comment]
@patrick-wright Yeah, we were discussing when we implement parallelization only showing the `acceptance.log` on stdout by default and then having Jenkins grab all the other logs for display or download asynchronously

**Comment by patrick-wright** [comment]
I'm :+1: on the design.

**Pull request merged**